openebench/APAeval_schemas/Challenges/APAeval_challenge1.json

```diff
@@ -0,0 +1,32 @@
+{
+    "_id": "APAeval:2021-06-04_Q2",
+    "_schema": "https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Challenge",
+    "name": "Correlation benchmark for RNA-seq poly(A) site quantification.",
+    "acronym": "Q2",
+    "benchmarking_event_id": "APAeval:2021-06-04",
+    "is_automated": true,
+    "dates": {
+        "creation": "2021-06-04T00:00:00Z",
+        "modification": "2021-06-04T00:00:00Z"
+    },
+    "challenge_contact_ids": [
+        "Alexander.Kanitz",
+        "Mervin.M.Fansler"
+    ],
+    "metrics_categories": [
+        {
+            "category": "assessment",
+            "description": "Metrics used to benchmark the correlation between RNAseq-based poly(A) site quantification and poly(A)-sites quantification based on orthogonal 3'end seq data.",
+            "metrics": [
+                {
+                    "metrics_id": "APAeval:corr_coeff",
+                    "tool_id": "APAeval:Tool_QAPA.json"
```

Comment:
If the examples in the repo below are correct, then tool_id would refer to a tool computing the metric, and a challenge would be comparing one or more metrics for a given dataset, but I don't know if it's a fixed rule or we have some freedom here.  
https://github.com/inab/benchmarking-data-model/tree/master/prototype-data/1.0.x/TCGA/Challenges

Reply:
Yes, I think tool_id should point to a summary workflow, like the one we have now that computes the correlations. I will try to work on this and to improve, but not sure how far I get.